app/Services/AdminUserServices/UserService.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace App\Services\AdminUserServices;
+
+use App\Enums\GeneralEnums;
+use App\Exceptions\BadRequestException;
+use App\Exports\GeneralReportExport;
+use App\Mail\Company\ClientOnboardingEmail;
+use App\Models\Company;
+use App\Models\Role;
+use App\Models\User;
+use App\Responser\JsonResponser;
+use App\Services\RoleServices\RoleService;
+use Carbon\Carbon;
+use Illuminate\Support\Facades\Auth;
+use Maatwebsite\Excel\Facades\Excel;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Mail;
+use Illuminate\Support\Str;
+
+class UserService
+{
+
+    protected RoleService $roleService;
+
+    public function __construct(RoleService $roleService)
+    {
+        $this->roleService = $roleService;
+    }
+
+    public function overview($request)
+    {
+        $currentUser = Auth::user();
+        $dateFilter = $request->date_filter;
+
+        if ($dateFilter === "1day") {
```

Comment:
Call GeneralHelper::dateFilter($request->date_filter) here. Anyone wey no dey the if block, add am